keep/providers/bigquery_provider/bigquery_provider.py

```diff
@@ -21,15 +21,15 @@ class BigqueryProviderAuthConfig:
     BigQuery authentication configuration.
     """
 
-    credentials_path: Optional[str] = dataclasses.field(
+     service_account_json: Optional[str] = dataclasses.field(
         default=None,
         metadata={
             "required": False,
-            "description": "Path to the service account key in JSON format. "
+            "description": "Path to the service account key file in JSON format. "
             "If not provided, will use application default credentials",
         },
     )
```

Comment:
I just took a quick look at GkeProviderAuthConfig, seems like the metadata should include something like this:
```python
    service_account_json: str = dataclasses.field(
        metadata={
            "required": True,
            "description": "The service account JSON with container.viewer role",
            "sensitive": True,
            "type": "file",
            "name": "service_account_json",
            "file_type": ".json",  # this is used to filter the file type in the UI
        }
    )
```

Reply:
yeah, it is working locally : )